### ISSUE TITLE
Public SSH Key Error

### ISSUE THREAD

**Issue by kikoking1** [morassman/atom-commander#65]
When a public SSH key is added to my mac computer, atom commander tries to use it whether I want it to or not through sftp connecting. It says that the encoding is too long, no matter which settings I use in the sftp. I had to remove my public SSH key from my mac to get it to work again. Is there a way to get it not to use the Public SSH key if one is on the computer, or make it so it doesn't error out with the encoding being too long?

**Comment by morassman** [comment]
Hi @kikoking1 

It isn't supposed to try and use the public key if it isn't configured to do so. I took a look at the code, but it isn't obvious how this can be happening. I'll experiment some more to see if I can reproduce this behaviour.

Just to make sure. So even if you choose 'Login with password' instead of 'Login with private key' you still get the error?

**Comment by kikoking1** [comment]
Yes, I get the same error even if I try to login with the private key and use the passphrase.

**Comment by morassman** [comment]
That makes sense, but do you get the error if you _don't_ choose 'Login with private key', but choose 'Login with password'?

<img width="472" alt="screen shot 2017-03-15 at 13 45 52" src="https://cloud.githubusercontent.com/assets/13697130/23947086/d4e8dd06-0985-11e7-8caf-6a329247c5d1.png">

**Comment by kikoking1** [comment]
Oh sorry, yes the error happens even if I choose Login with password.

**Comment by morassman** [comment]
I couldn't reproduce the issue myself, but I think I found it. I just released v0.8.8. Let me know if it fixed the problem.

**Comment by kikoking1** [comment]
Hey, so it looks like everything is working great now! I did get a weird error when I try to edit, or add a new, sftp account without putting an ssh key (When having one installed on my computer).
![screen shot 2017-03-19 at 9 30 08 pm](https://cloud.githubusercontent.com/assets/25890884/24088017/44ba9fa8-0ceb-11e7-8920-3f0f03fcd2d9.png)
- 

but I clicked ok anyway and it stilled logged me in no problem. 

It logs me in and test button returns connected successfully when I use the ssh key and passphrase.

**Comment by morassman** [comment]
Ok great. I'll look into the issue with testing the connection.
